### PR TITLE
[devtools-translate] Russian - What's New in DevTools (Chrome 100)

### LINKED ISSUE #2269: [devtools-translate] Russian - What's New in DevTools (Chrome 100)
Post: https://developer.chrome.com/blog/new-in-devtools-100/

Translator: @solarrust
Reviewers: @webmaxru, @kateryna-prokopenko

Translated by: Mar 22

### PR DESCRIPTION
**Pull request by solarrust**
Fixes #2269 

@jecfish I'm so glad to be back 🤍

@kateryna-prokopenko @webmaxru I had some doubts:
1. The part about Partially presented frames in the Performance panel. Please, check my translation closely. I'm not sure that understand all clearly.  
2. How would you translate «user agent» into Russian? I used «юзер агент» but I don't like it. 

Thank you! 🕊

### CODE REVIEW

**Comment by jecfish** [comment]
Glad that you are back! 💜

File: site/ru/blog/new-in-devtools-100/index.md [hunk 1]
```diff
@@ -65,69 +75,90 @@ Chromium issues: [1222574](https://crbug.com/1222574), [1222573](https://crbug.c
...
 
 <!-- You can now rename a recording in the [Recorder](/docs/devtools/recorder/) panel with the edit button (pencil icon) next to the recording’s title. -->
+Теперь вы можете переименовать запись на вкладке [Recorder](/docs/devtools/recorder/) нажав на кнопку редактирования (иконка карандаша) рядом с названием записи.
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/Pn9Xsrq9lnStmtjpe0jt.png", alt="Rename a recording", width="800", height="502" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/Pn9Xsrq9lnStmtjpe0jt.png", alt="Переименование записи", width="800", height="502" %}
 
 
 <!-- ## Preview class/function properties on hover {: #properties } -->
+## Предпросмотр свойств класса/функции по наведению курсора {: #properties }
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-100/index.md:113]:
```suggestion
## Предпросмотр свойств класса/функции при наведении курсора {: #properties }
```

File: site/ru/blog/new-in-devtools-100/index.md [hunk 2]
```diff
@@ -65,69 +75,90 @@ Chromium issues: [1222574](https://crbug.com/1222574), [1222573](https://crbug.c
...
 
 <!-- You can now hover over a class or function in the **Sources** panel during debugging to preview its properties. Previously, it only showed the function name and a link to its location in the source code. -->
+Теперь во время отладки вы можете во вкладке **Источники** (Sources) навести курсор мыши на класс или функцию чтобы посмотреть их свойства. Раньше показывалось только имя функции и ссылка на её расположение в исходном коде.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-100/index.md:116]:
```suggestion
Теперь во время отладки вы можете навести курсор мыши во вкладке **Источники** (Sources) на класс или функцию, чтобы посмотреть их свойства. Раньше показывалось только имя функции и ссылка на её расположение в исходном коде.
```

File: site/ru/blog/new-in-devtools-100/index.md [hunk 3]
```diff
@@ -65,69 +75,90 @@ Chromium issues: [1222574](https://crbug.com/1222574), [1222573](https://crbug.c
...
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/0a585b3883ad39f2f83fa5ab9c7731270d3a2974 ​#}
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/96fb7872ce01eb3fde267e39987a82ce3d3f3e21 #}
 
-Chromium issue: [1049947](https://crbug.com/1049947)
+Задача в трекере Chromium: [1049947](https://crbug.com/1049947)
 
 
 <!-- ## Partially presented frames in the Performance panel {: #perf } -->
+## Частично представленные кадры во вкладке Производительность (Performance) {: #perf }
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-100/index.md:127]: I've opened DevTools in Russian and it seems the timeline is called "Фреймы" and not "Кадры". Could you please change it everywhere in this passage?

File: site/ru/blog/new-in-devtools-100/index.md [hunk 4]
```diff
@@ -65,69 +75,90 @@ Chromium issues: [1222574](https://crbug.com/1222574), [1222573](https://crbug.c
...
 
 <!-- Performance recording now displays a new frame category "Partially presented frames" in the **Frames** timeline.  -->
+На запись производительности на временной шкале **Кадры** (Frames) теперь отображается новая категория кадров – «Частично представленные кадры».
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-100/index.md:130]:
```suggestion
На записи производительности на временной шкале **Фреймы** (Frames) теперь отображается новая категория кадров – «Частичный показ фрейма». 
```

File: site/ru/blog/new-in-devtools-100/index.md [hunk 5]
```diff
@@ -65,69 +75,90 @@ Chromium issues: [1222574](https://crbug.com/1222574), [1222573](https://crbug.c
...
 
 
 <!-- ## Partially presented frames in the Performance panel {: #perf } -->
+## Частично представленные кадры во вкладке Производительность (Performance) {: #perf }
 
 <!-- Performance recording now displays a new frame category "Partially presented frames" in the **Frames** timeline.  -->
+На запись производительности на временной шкале **Кадры** (Frames) теперь отображается новая категория кадров – «Частично представленные кадры». 
 
 <!-- Previously, the **Frames** timeline visualizes any frames with delayed main-thread work as "dropped frames". However, there are cases where some frames may still produce visual updates (e.g. scrolling) driven by the compositor thread. -->
+Раньше на временной шкале **Кадры** (Frames) любые кадры с отложенной работой по основному потоку отображались как «выброшенные кадры». Однако есть случаи, когда некоторые кадры могут продолжать производить визуальные обновления (например, при скролле), управляемые потоком компановки.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-100/index.md:133]:
```suggestion
Раньше на временной шкале **Кадры** (Frames) любые кадры с отложенной работой по основному потоку отображались как «отмененные фреймы». Однако есть случаи, когда некоторые кадры могут продолжать производить визуальные обновления (например, при скролле), управляемые потоком компоновки.
```

File: site/ru/blog/new-in-devtools-100/index.md [hunk 6]
```diff
@@ -65,69 +75,90 @@ Chromium issues: [1222574](https://crbug.com/1222574), [1222573](https://crbug.c
...
 
 <!-- This leads to user confusion because the screenshots of these “Dropped frames” are still reflecting visual updates.  -->
+Это может смутить пользователя, поскольку на скриншотах таких «выброшенных кадров» по прежнему видны визуальные изменения.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-100/index.md:136]:
```suggestion
Это может смутить пользователя, поскольку скриншоты таких «отмененных фреймов» по прежнему отображают визуальные изменения.
```

File: site/ru/blog/new-in-devtools-100/index.md [hunk 7]
```diff
@@ -65,69 +75,90 @@ Chromium issues: [1222574](https://crbug.com/1222574), [1222573](https://crbug.c
...
 
 <!-- The new "Partially presented frames" aims to indicate more intuitively that although some content is not presented timely in the frame, but the issue is not so severe as to block visual updates altogether. -->
+Новый тип «Частично представленные кадры» нужен чтобы показать более интуитивно, что хотя некоторое содержимое не представлено в кадре прямо сейчас, но проблема не настолько серьезная, чтобы полностью блокировать визуальные обновления.
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/QcqjnFhMz1Bxd5dkmduj.png", alt="Partially presented frames in the Performance panel", width="800", height="531" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/QcqjnFhMz1Bxd5dkmduj.png", alt="Частично представленные кадры во вкладке Производительность (Performance)", width="800", height="531" %}
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-100/index.md:141]:
```suggestion
{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/QcqjnFhMz1Bxd5dkmduj.png", alt="Частичный показ фрейма во вкладке Производительность (Performance)", width="800", height="531" %}
```

File: site/ru/blog/new-in-devtools-100/index.md [hunk 8]
```diff
@@ -65,69 +75,90 @@ Chromium issues: [1222574](https://crbug.com/1222574), [1222573](https://crbug.c
...
+Раньше на временной шкале **Кадры** (Frames) любые кадры с отложенной работой по основному потоку отображались как «выброшенные кадры». Однако есть случаи, когда некоторые кадры могут продолжать производить визуальные обновления (например, при скролле), управляемые потоком компановки.
 
 <!-- This leads to user confusion because the screenshots of these “Dropped frames” are still reflecting visual updates.  -->
+Это может смутить пользователя, поскольку на скриншотах таких «выброшенных кадров» по прежнему видны визуальные изменения.
 
 <!-- The new "Partially presented frames" aims to indicate more intuitively that although some content is not presented timely in the frame, but the issue is not so severe as to block visual updates altogether. -->
+Новый тип «Частично представленные кадры» нужен чтобы показать более интуитивно, что хотя некоторое содержимое не представлено в кадре прямо сейчас, но проблема не настолько серьезная, чтобы полностью блокировать визуальные обновления.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-100/index.md:139]:
```suggestion
Новый тип «Частичный показ фрейма» нужен чтобы показать более интуитивно, что хотя некоторое содержимое не представлено в кадре своевременно, проблема не настолько серьезна, чтобы полностью блокировать визуальные обновления.
```

File: site/ru/blog/new-in-devtools-100/index.md [hunk 9]
```diff
@@ -65,69 +75,90 @@ Chromium issues: [1222574](https://crbug.com/1222574), [1222573](https://crbug.c
...
 
 
 <!-- ## Miscellaneous highlights {: #misc } -->
+## Другие важные моменты {: #misc }
 
 <!-- These are some noteworthy fixes in this release: -->
+Вот несколько заслуживающих внимания исправлений в этом выпуске:
 
 <!-- - Updated iPhone user agent strings for [emulated devices](/docs/devtools/device-mode/#device). All iPhone versions after 5 have a user-agent string with iPhone OS 13_2_3. ([1289553](https://crbug.com/1289553)) -->
+- Обновили строку юзер агента для iPhone в [режиме эмуляции](/docs/devtools/device-mode/#device). Все iPhone с версией младше 5 теперь имеют юзер агент iPhone OS 13_2_3. ([1289553](https://crbug.com/1289553))
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-100/index.md:155]:
```suggestion
- Обновили строки юзер агента для [режима эмуляции](/docs/devtools/device-mode/#device). Все iPhone с версией новее 5 теперь имеют юзер агент строку iPhone OS 13_2_3. ([1289553](https://crbug.com/1289553))
```

File: site/ru/blog/new-in-devtools-100/index.md [hunk 10]
```diff
@@ -65,69 +75,90 @@ Chromium issues: [1222574](https://crbug.com/1222574), [1222573](https://crbug.c
...
 <!-- - You can now save [snippet](/docs/devtools/javascript/snippets/) as a JavaScript file directly. Previously, you needed to append `.js` file extension manually. ([1137218](https://crbug.com/1137218)) -->
+- Теперь вы можете сохранить [сниппет](/docs/devtools/javascript/snippets/) как JavaScript файл. Раньше нужно было вручную указывать файлу расширение `.js`. ([1137218](https://crbug.com/1137218))
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-100/index.md:157]:
```suggestion
- Теперь вы можете сохранить [сниппет](/docs/devtools/javascript/snippets/) как JavaScript файл. Раньше нужно было вручную добавлять файлу расширение `.js`. ([1137218](https://crbug.com/1137218))
```

File: site/ru/blog/new-in-devtools-100/index.md [hunk 11]
```diff
@@ -65,69 +75,90 @@ Chromium issues: [1222574](https://crbug.com/1222574), [1222573](https://crbug.c
...
 <!-- - The **Sources** panel now correctly displays scope variable names when debugging with sourcemap. Previously, the **Sources** panel displays minified scope variable names despite sourcemap being provided. ([1294682](https://crbug.com/1294682))  -->
+- На вкладке **Источники** (Sources) теперь корректно отображаются имена переменных области видимости при отладке с картой источников. Ранее на панели **Источники** (Sources) отображались минифицированные имена переменных области видимости, несмотря на предоставленную карту источников. ([1294682](https://crbug.com/1294682)) 
 <!-- - The **Sources** panel now restores scroll position correctly on page load. Previously, the position was not restored correctly causing inconvenience in debugging. ([1294422](https://crbug.com/1294422))  -->
+- Панель **Источники** (Sources) теперь корректно восстанавливает положение прокрутки при загрузке страницы. Ранее положение восстанавливалось некорректно, что вызывало неудобства при отладке. ([1294422](https://crbug.com/1294422))
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-100/index.md:161]:
```suggestion
- Вкладка **Источники** (Sources) теперь корректно восстанавливает положение прокрутки при загрузке страницы. Ранее положение восстанавливалось некорректно, что вызывало неудобства при отладке. ([1294422](https://crbug.com/1294422)) 
```

**Review by kateryna-prokopenko — COMMENTED** [review]
@solarrust thanks for the translation! Added some comments

File: site/ru/blog/new-in-devtools-100/index.md [hunk 12]
```diff
@@ -6,55 +6,65 @@ authors:
...
 <!-- end: translation instructions -->
 
 <!-- ## Chrome 100  {: #m100 } -->
+## Chrome 100  {: #m100 }
 
 <!-- Here’s to the 100th Chrome version! Chrome DevTools will continue to provide reliable tools for developers to build on the web. Take a moment to click around in the **What’s New** tab to celebrate the milestones. -->
+Сотая версия Chrome! Chrome DevTools будет продолжать предоставлять разработчикам надежные инструменты для создания веба. Уделите пару минут, чтобы пролистать вкладку **Что нового** и отпразновать важные вехи.
 
 <!-- As usual, you can watch the latest [What’s New in DevTools video](https://goo.gle/devtools-youtube) by clicking on the image. -->
+Как всегда вы можете узнать о последних изменениях из [видео What’s New in DevTools](https://goo.gle/devtools-youtube). Кликните на картинку.
```
[the remark this answers is not in the document]
Comment (solarrust) [reply site/ru/blog/new-in-devtools-100/index.md:39]: I think «новейшее» sounds pathos. May be «последнее» or «новое»?

**Comment by solarrust** [comment]
@kateryna-prokopenko thank you for the review! I fixed almost everything. Please, look it one more time 🙏

**Comment by kateryna-prokopenko** [comment]
Looks good to me, thanks! Replied regarding "новейшее" in the thread

**Review by jecfish — APPROVED** [review]

**Comment by jecfish** [comment]
Спасибо вам всем!

**Comment by jecfish** [comment]
Yeap, how about sent. PR?

**Pull request merged**